src/compiler/utilities.ts

```diff
@@ -744,34 +743,6 @@ export function getFullWidth(node: Node) {
     return node.end - node.pos;
 }
 
-/** @internal */
-export function getResolvedModule(sourceFile: SourceFile | undefined, moduleNameText: string, mode: ResolutionMode): ResolvedModuleFull | undefined {

```

Comment:
I'm honestly a little nervous to remove these APIs even though they're internal; people appear to be using it, including Roblox's TS-to-Lua converter and @andrewbranch's arethetypeswrong: https://github.com/search?q=%2Fts%5C.getResolvedModule%2F&type=code

Reply:
It also seems to be the case that (at least for ATTW) that some people don't actually have Programs at this point, either...

Reply:
With this change this can not work without giving program so i dont see what we can do here. Was internal for a purpose there

Reply:
Roblox appears to have a `Program` at that point, so that one is at least fixable (and a few other smaller examples). `dukat` doesn't in all uses, nor does ATTW.

Reply:
The properties being retrieved here would not exist if there were not a Program already. In my case the Program is somewhere else but I can get to it. In fact, I think what I’m doing becomes easier if it’s moved to the Program.

Reply:
Ah, you're totally right, I didn't realize! Consider this resolved 😄